src/components/Disclaimer/Disclaimer.scss

```diff
@@ -0,0 +1,45 @@
+#backButton {
+	float: left;
+	width: 10vw;
+	height: 10vh;
+	font-size: 400%;
+	transform: translate(-380px, 5px);
+	position: absolute;
+	background-color: #0a3d5f;
+	border: none;
+	-webkit-text-stroke: 9px #0a3d5f;
+}
+
+#bigCameraIcon {
+	padding-top: 144px;
+	transform: translate(-20px);
+	font-size: 400%;
+}
+
+#lockIcon {
+	padding-top: 96px;
+	transform: translate(-13px);
+	font-size: 400%;
+}
+
+.infoText {
```

Comment:
You should really use either all classnames or all Ids, unless there is a reason this is different specifically?